Makefile

```diff
@@ -28,11 +28,21 @@ RANLIB = ranlib
 
 CPPFLAGS = -I.
 # TODO: probably update cram code to make it compile cleanly with -Wc++-compat
-CFLAGS   = -g -Wall -O2
+CFLAGS   = -g -Wall -Wno-unused-function -O2
```

Comment:
I wonder whether instead of doing this it might be better to add __attribute((unused)) to the #defined functions from k*